src/routes/patterns/components/FormFields.js

```diff
@@ -1,7 +1,75 @@
 import React from "react";
+import { Card, CodeBlock, FormGroup, TextArea, Input, Subheading } from "styled";
 
-import { Card } from "styled";
+const FormFields = () => (
+  <Card mb={24} padded>
+    <Subheading mb={24}>Form Fields</Subheading>
 
-const FormFields = () => <Card />;
+    <Subheading capitalize color micro mb={8}>Form Group</Subheading>
+    <FormGroup stacked>
+      <Input value="Echo Foxtrot" mb={16} readOnly />
+      <TextArea value="Orange Pudding Queenie Robert Sugar Tommy Uncle Vinegar Willie Xerxes Yellow Zebra Nuts Orange Pudding. Charlie Duff Edward Freddy George Harry Ink Johnnie King London Monkey Nuts." readOnly height={77} />
+    </FormGroup>
+
+    <CodeBlock>
+{`import { Input } from "styled";
+
+const FormFields = () => (
+  <FormGroup>
+    {children}
+  </FormGroup>
+);
+
+// Props
+// readOnly     Turns field into read only
+// stacked      Stack content vertically
+`}
+    </CodeBlock>
+
+    <Subheading capitalize color micro mb={8}>Text Area</Subheading>
+    <TextArea placeholder="Enter text" />
```

Comment:
TextArea should autogrow ideally. It seems weird that it overflows after 2 lines

Reply:
https://github.com/andreypopp/react-textarea-autosize

Reply:
I didn't think this was a huge deal since these are primarily used on the draggable image cards and I wanted these to retain a consistent size.